phing/tasks/setup.xml

```diff
@@ -206,6 +206,16 @@
   </target>
 
   <target name="setup:update" description="Update current database to reflect the state of the Drupal file system.">
+    <!-- Enable features module. -->
+    <drush command="en" assume="yes" alias="${drush.alias}">
+      <param>features</param>
```

Comment:
We don't actually have the features module in our composer.json. Will this cause a failure if features doesn't exist? Perhaps we should add it to template/composer.json.